src/main/resources/database/changes/release-10.49.0/add_notes_to_sample_summary.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE sample.samplesummary ADD COLUMN notes VARCHAR(255);
```

Comment:
As above, would make this 250 to standardise

Reply:
I suppose ..